website/app/main.js

```diff
@@ -395,7 +395,10 @@
     // #146 Fix for vertical scrollbar appearing when mouse plugged in
     var css = document.createElement('style');
     css.type = 'text/css';
+
     var style = 'body { overflow: hidden; }'
+    style += '@media (max-width: 700px) { [aria-label="New Message"] { visibility: hidden; } }'
```

Comment:
This will only work for people with en_US locale on Facebook. Not even en_EN (where it's most likely "New message".) You might want to consider a selector (or some other way of finding the element) that doesn't rely on user settings. You can try things out by chaning your language under account settings on Facebook.